### ISSUE TITLE
UDPClientTest.BeginSend_AsyncOperationCompletes_Success failed in CI on Windows

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#3447]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_windows_debug_prtest/4229/console
```
Discovering: System.Net.Sockets.APMServer.Tests
  Discovered:  System.Net.Sockets.APMServer.Tests
  Starting:    System.Net.Sockets.APMServer.Tests
     System.Net.Sockets.Tests.UDPClientTest.BeginSend_AsyncOperationCompletes_Success [FAIL]
        System.Net.Sockets.SocketException : An address incompatible with the requested protocol was used
        Stack Trace:
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Net.Sockets\src\System\Net\Sockets\Socket.cs(3211,0): at System.Net.Sockets.Socket.DoBeginSendTo(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, EndPoint endPointSnapshot, SocketAddress socketAddress, OverlappedAsyncResult asyncResult)
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Net.Sockets\src\System\Net\Sockets\Socket.cs(3156,0): at System.Net.Sockets.Socket.BeginSendTo(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, EndPoint remoteEP, AsyncCallback callback, Object state)
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Net.Sockets\src\System\Net\Sockets\UDPClient.cs(598,0): at System.Net.Sockets.UdpClient.BeginSend(Byte[] datagram, Int32 bytes, IPEndPoint endPoint, AsyncCallback requestCallback, Object state)
           d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Net.Sockets.Legacy\tests\FunctionalTests\UdpClientTest.cs(51,0): at System.Net.Sockets.Tests.UDPClientTest.BeginSend_AsyncOperationCompletes_Success()
  Finished:    System.Net.Sockets.APMServer.Tests
```

**Comment by CIPop** [comment]
/cc: @josguil 
Temporarily disabling this.

**Comment by CIPop** [comment]
Improved/fixed by 2d6de874008b0ba74c877c5895847421f3adf1af.
